native/cocos/application/BaseGame.cpp

```diff
@@ -41,12 +42,18 @@ int BaseGame::init() {
                                                       cc::ISystemWindow::CC_WINDOW_INPUT_FOCUS
                                                 : _windowInfo.flags;
     std::call_once(_windowCreateFlag, [&]() {
-        if (_windowInfo.x == -1 || _windowInfo.y == -1) {
-            createWindow(_windowInfo.title.c_str(), _windowInfo.width, _windowInfo.height, _windowInfo.flags);
-        } else {
-            createWindow(_windowInfo.title.c_str(),
-                         _windowInfo.x, _windowInfo.y, _windowInfo.width, _windowInfo.height, _windowInfo.flags);
-        }
+        //if (_windowInfo.x == -1 || _windowInfo.y == -1) {
+        //    createWindow(_windowInfo.title.c_str(), _windowInfo.width, _windowInfo.height, _windowInfo.flags);
+        //} else {
+        //    createWindow(_windowInfo.title.c_str(),
+        //                 _windowInfo.x, _windowInfo.y, _windowInfo.width, _windowInfo.height, _windowInfo.flags);
+        //}
+
+        ISystemWindowManager *windowMgr = BasePlatform::getPlatform()->getInterface<ISystemWindowManager>();
+        ISystemWindow* window1 = windowMgr->createWindow("w1");
+        window1->createWindow(_windowInfo.title.c_str(), _windowInfo.width, _windowInfo.height, _windowInfo.flags);
```

Comment:
Has both `ISystemWindowManager::createWindow()` and `ISystemWindow::createWindow()` is strange.

Reply:
maybe ISystemWindow::create() ?

Reply:
Why just use `ISystemWindowManager::createWindow()` to create a real window?